### DISCUSSION TITLE
How do I deploy static files of the main branch such as index CSS JS？

### DISCUSSION THREAD

**Discussion by 12dxxb** [crazy-max/ghaction-github-pages#206] [Q&A] [answered]
I found that I could only write HTML in the YML file, and I could not use the file in my main branch directly

**Comment by crazy-max** [comment, marked as answer]
@15b883 If you talk about the [first workflow](https://github.com/crazy-max/ghaction-github-pages#workflow) in the README it's a basic example to deploy a simple `index.html` file. You can deploy to GitHub pages whatever you want. More info: https://pages.github.com/ and https://docs.github.com/en/github/working-with-github-pages
